lib/engines/content_object_store/app/views/content_object_store/content_block/documents/show.html.erb

```diff
@@ -12,6 +12,30 @@
       ),
     ) %>
 
+<div class="govuk-grid-row govuk-!-padding-top-8">
+  <div class="govuk-grid-column-full">
+    <!-- TODO: The prototype wants to show a counter beneath the title but
+    before the table, the existing component doesn't support that so I've left
+    it out for now-->
+    <%= render(
+      ContentObjectStore::ContentBlock::Document::Show::LinkedEditionsTableComponent.new(
+        caption: "Content appears in",
+        linked_content_items: @linked_content_items,
+      ),
+    ) %>
+
+    <% if @page_data.total_pages > 1 %>
```

Comment:
should this be part of the LinkedEditionsTable? 
Should we have some testing around pagination as well? It's a bit hard to manually test without creating a bunch of links... (perhaps seeded test data might be helpful at this point too?)

Reply:
Good idea! The pagination is tested in the Cucumber tests, but I think it's a good idea to roll the pagination into the component

Reply:
I'll merge now, but we can maybe look at improving this as part of https://trello.com/c/2sFJMLTO/364-show-what-pages-will-be-affected-by-a-content-change-before-confirming-the-change